src/services/verify/fragments.tsx

```diff
@@ -1,25 +1,49 @@
-import { isValid, VerificationFragment } from "@govtechsg/oa-verify";
-
-const revokeFragmentName = "OpenAttestationEthereumDocumentStoreRevoked";
-export const getNotRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment[] =>
-  fragments.filter((status) => status.name !== revokeFragmentName && status.status !== "SKIPPED");
-export const getRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment[] =>
-  fragments.filter((status) => status.name === revokeFragmentName && status.status !== "SKIPPED");
+import { isValid, VerificationFragment, utils } from "@govtechsg/oa-verify";
+import { TYPES } from "../../constants/VerificationErrorMessages";
 
 interface interpretFragmentsReturnTypes {
   hashValid: boolean;
   issuedValid: boolean;
   identityValid: boolean;
-  revokedValid: boolean;
 }
 
 // TODO: to handle all `skipped` scenario, probably a file format that is not .tt or .json
 export const interpretFragments = (fragments: VerificationFragment[]): interpretFragmentsReturnTypes => {
-  const notRevokeFragments = getNotRevokeFragment(fragments);
-  const revokeFragments = getRevokeFragment(fragments);
   const hashValid = isValid(fragments, ["DOCUMENT_INTEGRITY"]);
-  const issuedValid = isValid(notRevokeFragments, ["DOCUMENT_STATUS"]);
-  const revokedValid = revokeFragments.length === 0 || isValid(revokeFragments, ["DOCUMENT_STATUS"]);
+  const issuedValid = isValid(fragments, ["DOCUMENT_STATUS"]);
   const identityValid = isValid(fragments, ["ISSUER_IDENTITY"]);
-  return { hashValid, issuedValid, identityValid, revokedValid };
+  return { hashValid, issuedValid, identityValid };
+};
+
+export const errorMessageHandling = (fragments: VerificationFragment[]): string[] => {
```

Comment:
before interpreting the fragments, issue the various checks here (return early)
1. `isDocumentStoreAddressOrTokenRegistryAddressInvalid`
2. `contractNotFound`
3. `serverError`
4. `invalidArgument`

im not entirely convinced y we need to do these checks, unless at the creator side, we never check their config file at all. because before issuing if we perform those checks there (at creator side), we are guaranteed that the above checks would not even be needed.

Reply:
erm i understand that but in between creator and validation, user can do things to the file, users can also do things to it by either not using our creator or creating another creator that doesn't do the same thing as us that maybe update the fields in the document but not actually deploy a doc store or token registry. u know many possibility for malicious act.

Reply:
but if you are using our verifier, you need to conform to "our way" of the config file, otherwise it will not verify at all.
if you decide to spin off your own creator then they should also spin off their verifier as well.

what malicious act? can you give an example?

Reply:
like maybe some one get an old verifiable document and just change the values? or change the value of an unwrapped file and wrapped it after changing, we wont be able to get the correct error message right? so this checks is to get the exact error message.